common/properties/scriptMetadata.txt

```diff
@@ -123,13 +123,13 @@ Dupl; 33; 1BC20; FR; 1; EXCLUSION; NO; NO; NO; YES; NO
 Egyp; 33; 13153; EG; 3; EXCLUSION; NO; NO; YES; YES; NO
 Elba; 33; 10500; AL; 1; EXCLUSION; NO; NO; NO; NO; NO
 Elym; 33; 10FF1; IR; 1; EXCLUSION; YES; NO; NO; NO; NO
-Gara; 33; 10D5D; SN; 1; EXCLUSION; YES; NO; YES; NO; YES  # provisional data for future Unicode 16.0 script
+Gara; 33; 10D5D; SN; 1; EXCLUSION; YES; NO; YES; NO; YES
```

Comment:
No longer provisional since Unicode 16.0 is out now

Reply:
I would say to put the age here but that's kind of pointless because age data is a unicode property.